templates/components/itilobject/fields_panel.html.twig

```diff
@@ -244,6 +244,8 @@
 
             {{ include('components/itilobject/fields/global_validation.html.twig') }}
 
+            {{ fields.textField('externalid', item.fields['externalid'], __('External ID'), field_options) }}
```

Comment:
maybe a condition here, as you don't change the schema for changes and problems (fields_pnal twig file is shared between itilobjects)